.github/workflows/release.yml

```diff
@@ -21,9 +21,12 @@ permissions:
   contents: write
 
 jobs:
-  release:
+  create_release_branch:
     name: Create Release
-    runs-on: [self-hosted, release]
+    runs-on: ubuntu-latest
```

Comment:
This particular workflow does not need a special machine, so I set it to ubuntu-latest to avoid using a self-hosted machine when not necessary, but I can set to self-hosted if you think that makes more sense.